content/rs/administering/active-passive.md

```diff
@@ -177,3 +183,9 @@ which shard/s the command refers.
 
 The source and destination can have different shard counts and functions
 for placement of keys.
+
+### Syncer in Active-Passive Replication (Replica Of)
```

Comment:
I don't understand this paragraph. We need to discuss it

Reply:
What are your thoughts?

Reply:
Flush before full sync
When replication backlog is full.

In DR - before getting to work only from the replicaOf - disable the "replicaOf" / replication so we won't get flushed

Reply:
I addresses this in commit c08158ea1d702f028c307bfae0e3b3233e14bc22. What do you think?